src/attacks/square/square_attack.jl

```diff
@@ -7,10 +7,21 @@ include("utils.jl")
 
 # The black-box Square Attack developed by Andriuschenko et al. (https://link.springer.com/chapter/10.1007/978-3-030-58592-1_29)
 # The only free variable is the budget: iterations
-function SquareAttack(model, x, y, iterations; ϵ=0.3, p_init=0.8, min_label=0, max_label=9, verbose=false, clamp_range = (0, 1))
+function SquareAttack(
+    model,
+    x,
+    y,
+    iterations;
+    ϵ = 0.3,
+    p_init = 0.8,
+    min_label = 0,
+    max_label = 9,
+    verbose = false,
+    clamp_range = (0, 1),
+)
     Random.seed!(0)
     w, h, c = size(x)
-    n_features = c*h*w
+    n_features = c * h * w
 
     # Initialization (stripes of +/-ϵ)
     init_δ = rand(w, 1, c) .|> x -> x < 0.5 ? -ϵ : ϵ
```

Comment:
This is very specific to images, not sure how to address so will not use for testing (cannot be easily generalized)

Reply:
Yeah, square's implementation seems tailored to images. I followed the original paper in my impl